ld_ex2.py

```diff
@@ -0,0 +1,8 @@
+#input: list of integers, output: minimum element of that list 
+lst = [] #empty list 
+def min (lst): #required function
+    lst.sort() #sorting the given list (increased order) and then extracting the element in the first index
+    return(lst[0])
+
+if __name__ == "__main__":
+    print(lst[0])
```

Comment:
wrong use.
1. no list is defined in main.
2. lst[0] does not return the smallest element in a list.
3. i didnt even ask you to test your function, but if you want to, do it right.